### PR TITLE
Persons for leicester210

### PR DESCRIPTION
**Pull request by vitagrazia81**

### CODE REVIEW

File: new/PRS13947Eleskndros.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
+                  <persName xml:lang="gez" >እለ፡ እስክንድሮስ</persName>
+                  <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
+                  <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
+                  <persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>
+                    <faith type="Traditional">pagan</faith>
+                    <occupation type="ruler">Ruler during the Roman Empire</occupation>
```
Comment (eu-genia) [inline new/PRS13947Eleskndros.xml:54]: is he a historical person, also known from other traditions? 
In that case it should be either "Alexander" as `n1 `or at least
```
<persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
       <persName xml:lang="en" corresp="#n1" type="main">Alexander</persName>
```

delete `<persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>` this line makes no sense, `sameAs `should be used for wikidata ID (or other authority list)  inside of the `person `element
wikidata ID should be either found on wikidata if he is present, or an ID should be preferably created on wikidata
see https://betamasaheft.eu/Guidelines/?q=sameas&id=persons Example 4

File: new/PRS13947Eleskndros.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                    <birth/>
+                    <death/>
+                    <floruit>0303-0305</floruit>
```
Comment (eu-genia) [inline new/PRS13947Eleskndros.xml:57]: the right way to encode dates should be 
`<floruit notBefore="0303" notAfter="0305">Beginning of the 4th century; period of the 'Big Persecutions' against the Christians of the Roman Empire.</floruit>
`
are the dates correct?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13943BakweraSeyon.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">በኵረ፡ ጽዮን፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Bakʷǝra Ṣǝyon</persName>
+                    <faith type="EOTC"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit>mid-14th mid 15th century</floruit>
```
Comment (thea-m) [inline new/PRS13943BakweraSeyon.xml:54]: you could also additionally give the date in a searchable format: `<floruit notBefore="1350" notAfter="1449"/>` or similar

Comment (eu-genia) [inline new/PRS13943BakweraSeyon.xml:54]: `<floruit notBefore="1350" notAfter="1450">mid-14th mid 15th century</floruit>`

File: new/PRS13944NewayaMaryam.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="VP" when="2022-11-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ንዋየ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="Nǝwāya Māryām"/>
```
Comment (eu-genia) [inline new/PRS13944NewayaMaryam.xml:49]: `<persName xml:lang="gez" type="normalized" corresp="#n1">Nǝwāya Māryām</persName>`

File: new/PRS13944NewayaMaryam.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                    <faith type="EOTC"/>
+                    <nationality type="Ethiopia"></nationality>
+                    <birth/>
+                    <death/>
+                    <floruit>mid-14th mid 15th century</floruit>
```
Comment (eu-genia) [inline new/PRS13944NewayaMaryam.xml:54]: see above for floruit

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13947Eleskndros.xml (hunk 1 above)
Comment (vitagrazia81) [reply new/PRS13947Eleskndros.xml:54]: Thanks Zhenia. 

It is not known who this Alexander is,  he simply mentioned in the Gadl (also in the Greeck Passio) as  ruler  during the period of the Big Persecutions (reign of  Diocletian), i.e. 303-305, in the region of Cilicia (Asia Minor).  His historicity is unknown.
For the moment I cannot add information about this Alexander. 
I will correct.

Actually, here I wanted to indicate that there are two attested variants of the Geez name, ʾƎleskǝndros and ʾƎla ʾǝskǝndros, both corresponding to the English Alexander. How I did is the right way?

File: new/PRS13947Eleskndros.xml (hunk 2 above)
Comment (vitagrazia81) [reply new/PRS13947Eleskndros.xml:57]: yes, this is the period when the big persecutions from Dioceltian occured. If this the question, yes, it is correct.

File: new/PRS13947Eleskndros.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS13947Eleskndros.xml:54]: Yes but you need the English form as the translation to `n1 `with type `main`, as he is not Ethiopian, see the correction above

Comment (vitagrazia81) [reply new/PRS13947Eleskndros.xml:54]: Thanks.
I did as follows:

  <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
                    <persName xml:lang="en" corresp="#n1" type="main">Alexander</persName>
                    <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
                    <persName xml:lang="en" corresp="#n2" type="main">Alexander</persName>

File: new/PRS13943BakweraSeyon.xml [hunk 6]
```diff
@@ -51,7 +51,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <nationality type="Ethiopia"/>
                     <birth/>
                     <death/>
-                    <floruit>mid-14th mid 15th century</floruit>
+                    <floruit notBefore="1350" notAfter="14549">mid-14th mid 15th century</floruit>
```
Comment (eu-genia) [inline new/PRS13943BakweraSeyon.xml]: 1449 or 1450

File: new/PRS13947Eleskndros.xml [hunk 7]
```diff
@@ -47,15 +47,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
+                    <persName xml:lang="en" corresp="#n1" type="main">Alexander</persName>
                     <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
-                  <persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>
+                    <persName xml:lang="en" corresp="#n2" type="main">Alexander</persName>
```
Comment (eu-genia) [inline new/PRS13947Eleskndros.xml:53]: only one (`n1`) may be type="main", this is what the app must display, please remove the entire translation or at least type="main" here

File: new/PRS13947Eleskndros.xml [hunk 8]
```diff
@@ -47,15 +47,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                     <faith type="Traditional">pagan</faith>
-                    <occupation type="ruler">Ruler during the Roman Empire</occupation>
+                    <occupation type="ruler">Mentioned in the 'Gadla Qirqos' as a ruler in Cilicia (Asia Minor), during the period of the 'Big Persecutions' (303-305 AD.).</occupation>
                     <birth/>
                     <death/>
-                    <floruit>0303-0305</floruit>
-                    <note>Beginning of the 4th century; period of the 'Big Persecutions' against the Christians of the Roman Empire.</note>
+                    <floruit notBefore="0303" notAfter="0305"></floruit>
+                    <note>Beginning of the 4th century; period of the 'Big Persecutions' (303-305 AD.) against the Christians of the Roman Empire.</note>
```
Comment (eu-genia) [inline new/PRS13947Eleskndros.xml:59]: you do not need the `note`, you should insert this note content inside of `floruit `as suggested

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13943BakweraSeyon.xml (hunk 6 above)
Comment (vitagrazia81) [reply new/PRS13943BakweraSeyon.xml]: Yes, I was in doubt how to write. In the @thea-m suggestion, there is 1449.
Does the first hlf of the 15th century  corresponds to 1449 or 1450?

Comment (vitagrazia81) [reply new/PRS13943BakweraSeyon.xml]: * correspond

File: new/PRS13947Eleskndros.xml (hunk 8 above)
Comment (vitagrazia81) [reply new/PRS13947Eleskndros.xml:59]: Ah ok, Sorry.

File: new/PRS13947Eleskndros.xml (hunk 7 above)
Comment (vitagrazia81) [reply new/PRS13947Eleskndros.xml:53]: So, I will delete  the entire translation for n2. Is that correct?

Comment (eu-genia) [reply new/PRS13947Eleskndros.xml]: yes, that would be better

File: new/PRS13943BakweraSeyon.xml (hunk 6 above)
Comment (eu-genia) [reply new/PRS13943BakweraSeyon.xml]: these numbers are not visible, they are only useful for filtering down in searches
1350-1449 or 1350-1450 are both ok but 1449 is more precise if we say that 1450 is the beginning of the second half

**Review by eu-genia — APPROVED** [review]

File: new/PRS13943BakweraSeyon.xml (hunk 6 above)
Comment (vitagrazia81) [reply new/PRS13943BakweraSeyon.xml]: Ok, thanks.

File: new/PRS13943BakweraSeyon.xml (hunk 3 above)
Comment (vitagrazia81) [reply new/PRS13943BakweraSeyon.xml]: Thanks.

**Comment by eu-genia** [comment]
I had approved already

**Comment by vitagrazia81** [comment]
Yes, sorry, it was by mistake and I could not undo it.

**Review by thea-m — APPROVED** [review]

**Pull request merged**